os/arch/arm/src/artik053/src/artik053_tash.c

```diff
@@ -315,13 +319,20 @@ int board_app_initialize(void)
 			lldbg("ERROR: FAILED TO smart_initialize\n");
 			free(rambuf);
 		} else {
-			(void)mksmartfs(CONFIG_ARTIK053_RAMMTD_DEV_POINT, false);
-
-			ret = mount(CONFIG_ARTIK053_RAMMTD_DEV_POINT, CONFIG_ARTIK053_RAMMTD_MOUNT_POINT,
-						"smartfs", 0, NULL);
-			if (ret < 0) {
-				lldbg("ERROR: Failed to mount the SMART volume: %d\n", errno);
+#ifdef CONFIG_SMARTFS_MULTI_ROOT_DIRS
+			ret = mksmartfs(CONFIG_ARTIK053_RAMMTD_DEV_POINT, 1, false);
```

Comment:
How about below code?

ret = mksmartfs();
if (ret == OK) { 
      ret = mount();
}
if (ret < 0) {
   lldbg();
   free();
}
and, s5jt200_tash.c also needs code clear.

Reply:
I followed others smiliarly, Do you want to change all?
If these are changed as you said, I think it will be difficult to show messages about case exactly.

Reply:
Alright, I Accept your opinion.